projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -211,6 +211,14 @@ function wpcom_launchpad_get_task_list_definitions() {
 			),
 			'is_enabled_callback' => 'wpcom_launchpad_is_hosting_flow_enabled',
 		),
+		'subscribers'            => array(
+			'title'    => 'Subscribers',
+			'task_ids' => array(
+				'import_subscribers',
+				'share_site',
+				'add_subscribe_block',
```

Comment:
Shouldn't this be the other way around? I would imagine we want the subscribe block to be added to the site before we ask users to share the site. I know our tasks aren't blocking, but we should still try to sequence them in a way that's likely to produce better outcomes.
```suggestion
				'add_subscribe_block',
				'share_site',
```

Reply:
I agree. What do you think @crisbusquets ? This is about the order in which we display the tasks.

Reply:
I've commited the change. We can always revert this.